wasm/src/main/java/com/dylibso/chicory/wasm/Encoding.java

```diff
@@ -11,6 +13,126 @@ public final class Encoding {
 
     private Encoding() {}
 
+    static int readInt(ByteBuffer buffer) {
```

Comment:
Do these need to be public? Could they be needed for parsing custom sections?

This is just a question to consider. It's fine to leave them private for now.

Reply:
In the current status I think that there are quite a few changes necessary to offer a proper, usable and generic API to parse custom sections ...
I would not try to predict the future, and welcome effort in this direction when someone needs it.

Opening visibility is not going to be a breaking change.